segmentstore/server/src/test/java/io/pravega/segmentstore/server/SegmentStoreMetricsTests.java

```diff
@@ -196,6 +199,22 @@ public void testCacheManagerMetrics() {
         assertNull(MetricRegistryUtils.getTimer(MetricsNames.CACHE_MANAGER_ITERATION_DURATION));
     }
 
+    @Test
+    public void testThreadPoolMetrics() throws Exception {
+        @Cleanup("shutdown")
```

Comment:
Do we need to use `ExecutorServiceHelpers.shutdown` instead of `@Cleanup("shutdown")` here?

Reply:
see above. Actually `shutdownNow` would be preferred as it's a decisive termination where all executing threads are interrupted. `shutdown` is a soft termination where the currently executing tasks are "Returned" so that the caller may (or may not) do anything with them. 

For this particular test, it doesn't matter. For more complex tests where we actually use the executors, I prefer `@Cleanup("shutdownNow")` AND `ExecutorServiceHelpers.shutdown` (former for catch-all, latter for normal execution).